tests/hpc/enable_in_zypper.pm

```diff
@@ -29,7 +29,10 @@ sub run {
     assert_script_run "zypper lr | grep $reponame";
 
     zypper_call("--gpg-auto-import-keys ref");
-    zypper_call 'up';
+
+    # The source where we get ganglia from at the moment cannot be updated
+    # needs to be changed once ganglia is officially released
+    zypper_call 'up' if !check_var('HPC', 'ganglia_server') || !check_var('HPC', 'ganglia_client');
```

Comment:
This can be rewritten as if get_var('HPC') !~ /^(ganglia_server|ganglia_client)$/